understanding/20/reading-level.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Reading Level</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users can get a simplified version of complex information</dd>
-         <dt>Author task</dt><dd>When text information becomes complex, create a more easily understood version</dd>
+         <dt>Goal</dt><dd>Users can get a simplified version of complex information.</dd>
+         <dt>What to do</dt><dd>When text information becomes complex, create a more easily understood version.</dd>
+         <dt>Why it's important</dt><dd>Some people, including those with cognitive disabilities, may not understand the meaning of content.</dd> 
```

Comment:
```suggestion
         <dt>Why it's important</dt><dd>More people, especially those with cognitive disabilities, can understand the meaning of content.</dd> 
```

Reply:
I think this is a bit better "More people, especially those with cognitive disabilities, understand the meaning of the content." and perhaps easier to understand.